WooCommerce/WooCommerceTests/Tools/CurrencySettingsTests.swift

```diff
@@ -5,9 +5,22 @@ import Yosemite
 /// CurrencySettings Tests
 ///
 class CurrencySettingsTests: XCTestCase {
-    func testInitDefault() {
-        let moneyFormat = CurrencySettings()
 
+    var moneyFormat = CurrencySettings()
+    var siteSettings = [SiteSetting]()
+
+    override func setUp() {
```

Comment:
In theory, moneyFormat and siteSettings should be initialised in setup and nullified in tearDown.

But because almost all tests use a different moneyFormat, maybe it makes sense to just initialise siteSettings in setup(), and initialise moneyFormat as a local variable for each test?